package.json

```diff
@@ -14,18 +14,22 @@
   "devDependencies": {
     "@types/jest": "^29.5.1",
     "@types/node": "^18.15.12",
-    "aws-cdk": "2.99.1",
+    "aws-cdk": "2.122.0",
     "cdk-nag": "^2.27.186",
     "jest": "^29.5.0",
     "ts-jest": "^29.1.0",
     "ts-node": "^10.9.1",
     "typescript": "~5.0.4"
   },
   "dependencies": {
-    "@aws-quickstart/eks-blueprints": "1.12.0",
-    "aws-cdk-lib": "2.99.1",
-    "@aws-cdk/aws-lambda-python-alpha": "^2.39.0-alpha.0",
+    "@aws-quickstart/eks-blueprints": "1.13.1",
+    "aws-cdk-lib": "2.122.0",
+    "@aws-cdk/aws-lambda-python-alpha": "2.122.0-alpha.0",
     "constructs": "^10.2.1",
     "source-map-support": "^0.5.21"
+  },
+  "overrides": {
+    "aws-cdk-lib": "2.122.0",
```

Comment:
is it a temporary solution until the issue is fixed upstream? if so, could you please create an issue so we can remove it when conditions allow?